R/formula_list_generators.R

```diff
@@ -533,6 +533,7 @@ EulerMultinomialUpdateMethod = function(change_model) {
     if (!scalar_expr) {
       vec = sprintf("c%s", vec)
     }
+    if(!simple_expr & scalar_expr){vec = sprintf("c%s", vec)}
```

Comment:
Thanks @jfree-man .  Can you remind me of what a non-simple expression is?

Reply:
Strings that don't look like this "^[a-zA-Z0-9.]+$". The only non-simple strings I think that I have encountered are those containing underscores. Ex. the flow name "hospital_recovery" in the shiver model was causing an error when using `mp_hazard`.

Reply:
This makes me think that we should change the definition of simple.  I think that simple should include names with underscores.

Reply:
Okay, I can add the underscore to the simple expressions and do some testing.

Reply:
I agree. I hadn't considered formulas, I was only thinking about names of variables, flows, etc.

Reply:
Cool.  Looks like you just had a commit that fixes this?  Maybe we should also have a comment in the code to define what is meant by 'simple'?  Taking full responsibility for not having such a comment.  Thanks for tracking this down.

Reply:
Good idea, I will add this.